Dockerfile

```diff
@@ -0,0 +1,59 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM golang:1.14 as api-builder
+
+WORKDIR /go/src/app
+
+COPY ./api .
```

Comment:
where is the source codes from? No need to get the source code package first?

Reply:
source codes is in work directory, we can use tag to control source code, it is not need to get codes from release.
if we have a tag `v2.0-rc3`, it's directory as beflow
```
apisix-dashboard
+-api
+-web
+-Makefile
+-Dockerfile
```
when we run `docker build .` in this directory, docker will default copy all files to build stage, you can configure what file is need for image by `ADD stage-path container-path`.
so when we execute `docker build` in specific tag, it will use the specific  codes.

Reply:
If we want to have an independent `apisix-dashboard-docker` project, we need to download souce code package in docker.

Reply:
Let's download codes from the target release first, then Build & Deploy in multiple stages.

Reply:
Let me resolve this conversation first.